eng/pipelines/maintenance-packages.yml

```diff
@@ -76,7 +77,8 @@ stages:
           displayName: $(_BuildJobDisplayName)
 
       - ${{ if eq(variables._RunAsPublic, True) }}:
-        - job: linux
+        - job: build_linux
+          displayName: 'Build Linux'
```

Comment:
This is also running in the official build and I don't think it should.  We're disabling tests so we should also disable this leg which shouldn't produce packages.

Reply:
So in the official build we would only leave the windows one? I assume that if we enable packing, it would contain all the DLLs for all OSs/platforms. Correct?